### PR TITLE
Destroy connections

### PR DESCRIPTION
**Pull request by yonivy**
- Add a `destroy` function to close all the open connections
- Upgrade dependencies
- Add the `debug` package for nicer debugging output
- Format for consistency

**Note**: testing this in a unit test was problematic for me. I'm having a hard time situating a scenario that I can manage, so if anyone has an idea on how I can test the `destroy` functionality I'm all ears.

### CODE REVIEW

**Review by Offir — APPROVED** [review]
Nice.
1. We need to change the version in package.json https://github.com/panoplyio/dbstream-mongo/blob/58e1be343aff933877414cd2592b2be4c7917362/package.json#L3
Since dbstream-mongo is now under panoply should we change also the version to 1.0.0?

2. See small question below.

 Note: I don't have idea how to test the `destroy` functionality

File: mongo.js [hunk 1]
```diff
@@ -241,13 +247,29 @@ module.exports.connect = function( url, options ) {
...
 }
 
+module.exports.destroy = function() {
+    for ( const url in connections ) {
+        debug('closing', url);
+
+        clearTimeout( connections[ url ].closeTimeout )
+        delete connections[ url ].closeTimeout;
+
+        if( connections[ url ].client ) {
```
Comment (Offir) [inline mongo.js:257]: What will happen if there is more than 1 client on the same `connections[ url ]`?
https://github.com/panoplyio/dbstream-mongo/blob/58e1be343aff933877414cd2592b2be4c7917362/mongo.js#L142

Comment (yonivy) [reply mongo.js:257]: As far as I can tell there's only 1 actual client per `connections[ url ]` so closing it should do the trick. When a client is added all that is affected is a counter.

**Comment by yonivy** [comment]
regarding the package version well.. it's meaningless. We're working with git tags. In the base repo it does make sense because this package is published to npm.

**Pull request merged**